### ISSUE TITLE
Update automation API to enable/disable user accounts

### ISSUE THREAD

**Issue by dspruell** [MISP/MISP#2270]
The REST API currently supports user deletion:

> You can also delete users by POSTing to the below URL, but keep in mind that disabling users (by setting the disabled flag via an edit) is always prefered to keep user associations to events intact.

`https://<misp url>/admin/users/delete/[user id]`

Would it be useful to support a dedicated calls in API and PyMISP to disable and enable a user, or is it only preferred to edit the user to alter the `disabled` field?

**Comment by iglocska** [comment]
I think edit does a fine job at it, it's enough to just POST {"disabled":1} to get the user disabled, unless I am mistaken.

**Comment by dspruell** [comment]
That is correct. Editing a user to change the flag definitely gets the job done, although I was wondering if user enable/disable was enough of an important action to deserve e.g. it's own shortcut option in the UI and either a full fledged API call or a shortcut function in PyMISP for convenience/clarity. Deleting users isn't recommended in most cases because of potential impact to referential integrity, but both the API and UI offer the option prominently.

If it's not worth the hassle and the current method is good enough, I won't cry if this issue is just closed. :)  Thanks.

**Comment by iglocska** [comment]
We could do it, we did something similar with attribute deletion at one point. Originally an attribute delete always hard-deleted the attribute, now it sets the deleted flag, we could have a similar approach to the default action with a checkbox for a hard delete on the UI / an additional required parameter for the /delete API
